src/test/java/io/nats/client/AuthTests.java

```diff
@@ -88,6 +90,32 @@ private void assertEncoded(String encoded, int port) throws IOException, Interru
         c.close();
     }
 
+    @Test
+    @EnabledOnOs({ WINDOWS })
```

Comment:
I could not get this run on GH Actions. I'm assuming that it has something to do with how the command line handles control characters on unix systems.